src/webservice/GetMetricsHandler.cpp

```diff
@@ -0,0 +1,87 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "webservice/GetStatsHandler.h"
+#include "webservice/Common.h"
+#include "stats/StatsManager.h"
+#include <proxygen/lib/http/ProxygenErrorEnum.h>
+#include <proxygen/httpserver/ResponseBuilder.h>
+#include "GetMetricsHandler.h"
+
+namespace nebula {
+
+using proxygen::HTTPMessage;
+using proxygen::HTTPMethod;
+using proxygen::ProxygenError;
+using proxygen::UpgradeProtocol;
+using proxygen::ResponseBuilder;
+using nebula::stats::StatsManager;
+
+void GetMetricsHandler::onRequest(std::unique_ptr<HTTPMessage> headers) noexcept {
+    if (headers->getMethod().value() != HTTPMethod::GET) {
+        // Unsupported method
+        err_ = HttpCode::E_UNSUPPORTED_METHOD;
+        return;
+    }
+    serializer_ = DCHECK_NOTNULL(dynamic_cast<stats::MetricsSerializer*>(&StatsManager::get()));
```

Comment:
Need  dynamic_cast? we can assign subclass pointer to base class type directly.

Reply:
I think explicit cast better.

Reply:
The explicit `dynamic_cast` usage scenarios usually is to cast base type pointer to subclass pointer type.

Reply:
Ok, could change to static_cast label explicit casting.

Reply:
No need any `cast` here! This is language-level specified.